vultr-dns.py

```diff
@@ -1,43 +1,49 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Supporting Python 3 is one thing (like with the `print` fix, thanks!), but requiring it is another.

Python 2.7 is not quite dead; it's still the default Python in LTS distributions like Ubuntu Focal.

Is there a particular reason you had in mind for this change?

Reply:
Before PR, I have noticed it. There is only a `master` branch in this repository, however, I have no choice but to merge into it. 🤣 🤣 

Maybe we could make a new branch like `Python3` and update the `README` file?

BTW, since I am a newbie here, what should I do next? Creating a new PR after a new branch init?

Reply:
Well, with your changes, the program works in either Python 2 and 3, right?

So if we just leave `#!/usr/bin/env python` as it is, the program will use the default Python interpreter on the system, which may be Python 2 or Python 3 (it depends on the system).

To me, that feels like the right thing to do. I don't see any reason to maintain a separate branch.

Reply:
I have tested in my server, both python2 and 3 works fine. Just do as you say 😄 

Update `README` to tell users to change `#!/usr/bin/env python` according to their system's setting. Eg. I have to use `#!/usr/bin/env python3` to run this script by python3 and `#!/usr/bin/env python2` to runby python2.

Reply:
Hold on, I test my script using the command `certbot renew --dry-run`. I remember that using certbot to add a wildcard domain now needs to:
- add a `_acme-challenge` TXT record in the nameserver.
- add a record in `http://example@domain/.well-known/acme-challenge/xxxxxx_xxx-XXXXXX`.  

This script only includes the first one,  so **it is not correct and needs to add a new part.**

Reply:
Good to hear. 

We could also detect the situation where an HTTP challenge is present, and remind the user to include `--preferred-challenges dns`.

But, let's leave that for another time.